Sources/GRPC/ConnectionPool/GRPCChannelPool.swift

```diff
@@ -275,6 +275,10 @@ extension GRPCChannelPool.Configuration {
     /// Defaults to 100.
     public var maxWaitersPerEventLoop: Int = 100
 
+    /// The minimum number of connections to keep open in this pool, per EventLoop.
+    /// This number of connections per EventLoop will never go idle and be closed.
+    public var minConnections: Int = 1
```

Comment:
Defaulting to one is a semantic change, we should default to zero which is the current behaviour.